### ISSUE TITLE
Unable to edit access to specific levels

### ISSUE THREAD

**Issue by tomjones1977** [moloch--/RootTheBox#488]
I have created three levels, (0, 1, 2) and set level 1 and 2 as managed.  However, it appears I can't manually add/remove access to teams, when I hit save and open the edit box again, the team lists have gone back to the previous settings. 

I'd like to be able to add specific teams to different levels without them having to complete previous levels - is this possible with the `Managed` option?

**Comment by eljeffeg** [comment]
Yes, that's how it should work.  Are you getting any errors on the console logs?  I tried to test this myself and I couldn't replicate the issue.

**Comment by tomjones1977** [comment]
No errors in the container logs when trying to edit the membership for the level.  Can the log level be increased for the container?  I can try again that way.

**Comment by eljeffeg** [comment]
I started it up in the docker container and was able to reproduce the issue.  I'll investigate why that might be different.

**Comment by eljeffeg** [comment]
Pushed a fix - Thanks for the report

**Comment by tomjones1977** [comment]
Tested - confirmed working after a container image update.  Thank you for the rapid response!

**Comment by tomjones1977** [comment]
Is there somewhere to sponsor the project?  I couldn't see anything on the docs/wiki/homepage?
